README.md

```diff
@@ -21,4 +20,4 @@ Unless you explicitly state otherwise, any contribution intentionally submitted
 inclusion in Mayastor by you, as defined in the Apache-2.0 license, is licensed as above,
 without any additional terms or conditions.
 
-[![FOSSA Status](https://app.fossa.com/api/projects/custom%2B162%2Fgithub.com%2Fopenebs%2Fmayastor-extensions.svg?type=large&issueType=license)](https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2Fopenebs%2Fmayastor-extensions?ref=badge_large&issueType=license)
\ No newline at end of file
+[![FOSSA Status](https://app.fossa.com/api/projects/custom%2B162%2Fgithub.com%2Fopenebs%2Fmayastor-extensions.svg?type=large&issueType=license)](https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2Fopenebs%2Fmayastor-extensions?ref=badge_large&issueType=license)
```

Comment:
Didn't understand the change diff here.

Reply:
Yeah, I also did not not see what changed. I will probably ignore this.